docs/tutorials/google/advanced_calibration.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<p>worth opening an issue to support this function natively? namely: provide a list of metrics and generate a grid of plots. Note that if you add a third metric it looks rather squished. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>At least for tutorials I think this is much better than plotting them sequentially and having to scroll through everything. Not sure if useful outside of this, though.</p>